exporter/signalfxexporter/dpclient.go

```diff
@@ -104,7 +129,11 @@ func (s *sfxDPClient) pushMetricsDataForToken(
 		return numTimeseries, consumererror.Permanent(err)
 	}
 
-	req, err := http.NewRequest("POST", s.ingestURL.String(), body)
+	datapointURL := *s.ingestURL
+	if !strings.HasSuffix(datapointURL.Path, "v2/datapoint") {
+		datapointURL.Path = path.Join(datapointURL.Path, "v2/datapoint")
+	}
```

Comment:
I see, we append the suffix here. Was anything wrong with the previous implementation?

Reply:
Yes, since both datapoints and events use the same base ingest URL, we need a way to specify the base ingest server for both.  For backwards compatibility, we do still have to accept config with the path `/v2/datapoint` since previous configs might include that, but events are sent to `/v2/events`, so it is better to specify this config without the path in the YAML config.